Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerFetchTask.cpp

```diff
@@ -135,7 +135,8 @@ template<typename Message> bool ServiceWorkerFetchTask::sendToServiceWorker(Mess
 
 template<typename Message> bool ServiceWorkerFetchTask::sendToClient(Message&& message)
 {
-    return m_loader.connectionToWebProcess().connection().send(std::forward<Message>(message), m_loader.coreIdentifier()) == IPC::Error::NoError;
+    Ref loader = *m_loader;
```

Comment:
Is this style better/safer than our old style of:
`Ref loaderProtect(*m_loader);`

We used to use?

Reply:
Oh, we can call this protectedLoader. Will change.